.github/workflows/check-unlinked-content.js

```diff
@@ -0,0 +1,133 @@
+var fs = require("fs");
```

Comment:
Same check as applied in https://github.com/hashicorp/vault/pull/11391 - not sure this is a long-term solution, but better than nothing for now.

Rest of this PR is also the same approach as https://github.com/hashicorp/vault/pull/11391 👍